pymongo/topology.py

```diff
@@ -550,6 +556,9 @@ def _ensure_opened(self):
 
         Hold the lock when calling this.
         """
+        if self._closed:
+            raise InvalidOperation("Once a Topology is closed, "
+                                   "all operations raise an error")
```

Comment:
Let's come up with a more informative error message for the user.

Reply:
Tried my best to make it a bit more clear.